### ISSUE TITLE
Speed Up Not Showing After 10 Minutes

### ISSUE THREAD

**Issue by wjmelements** [MetaMask/metamask-extension#7054]
I have had a transaction pending for 12 minutes and I would like to increase its gas price, but there is no speed up button. I don't know the intended behavior but the button is usually available within 10 minutes.
<img width="355" alt="Screen Shot 2019-08-21 at 5 22 06 PM" src="https://user-images.githubusercontent.com/799573/63477136-a2cbf900-c438-11e9-99ad-8451efcc3d7f.png">

I typically override the default gas prices because even the slow setting is generally twice what EthGasStation recommends. Sometimes, I override it without checking the gas station first, and then want to speed up the transaction shortly after I sent it. Exposing the button to me sooner would be great.

MetaMask version 7.0.1

**Comment by wjmelements** [comment]
The button also did not show after restarting MetaMask.

**Comment by wjmelements** [comment]
I still do not see the button. Was the feature removed, perhaps?

**Comment by Gudahtt** [comment]
Was this the earliest pending transaction? Transactions have to be confirmed in the order of the transaction nonce. We only show the "Speed Up" button on the earliest pending transaction, as that one has to come first anyway.

**Comment by wjmelements** [comment]
> Was this the earliest pending transaction? Transactions have to be confirmed in the order of the transaction nonce. We only show the "Speed Up" button on the earliest pending transaction, as that one has to come first anyway.

It was the earliest pending transaction. It did not show. It was a straight ether transfer, no contract data. It wasn't the first transfer that session, but the prior one had confirmed.

**Comment by wjmelements** [comment]
I may have also had a pending transaction via another account.

**Comment by jernejml** [comment]
Same issue for me. There are no other pending transactions (and IIRC there weren't any other triggered). It's a contract transaction, 
so i doubt type of it has any effect. I made hundreds of (metamask) transactions before and this is the first time this occurred. 
Metamask version atm is 7.7.9, but it happened like a week ago.

**Comment by MicahZoltu** [comment]
Issue is present on 8.0.6 in Firefox.  I have a transaction with a low gas price that only shows cancel.  It has been sitting for about an hour and I only see cancel button still.

**Comment by brad-decker** [comment]
@MicahZoltu Firefox 8.0.6, did you have any other transactions ahead of that one? Could you possibly send a screenshot and state logs to https://metamask.zendesk.com/hc/en-us/requests/new so that we can look into this?

**Comment by MicahZoltu** [comment]
@brad-decker It was the only transaction in the queue.  I resolved my own issue by resetting my account and re-submitting the transaction, so I no longer have the ability to get a screenshot of the issue.  Would state logs still be available given I reset the account and have closed all related web pages?

**Comment by brad-decker** [comment]
@MicahZoltu I doubt your state logs would contain the erroneous state now. If it happens again please use the link to share screenshots and state logs. Also if anyone else comes to this issue with similar experience please do this before resetting your account. I'll attempt to reproduce. @MicahZoltu was this an eth transaction or a token send?

**Comment by MicahZoltu** [comment]
@brad-decker It was an ENS registration (second step).  I put in a very low amount of gas initially, but MM told me that the transaction was dropped (gas price too low).  I then put in a higher (but still low-ball) gas price which got picked up.  After waiting a while, I looked for the speedup button but was unable to find it.

**Comment by anastk** [comment]
I am dealing with the same issue but I couldn't solve it. 
I was interacting with a smart contract (Rarible NFT Minting), I noticed the option to speed up the transaction was there in MetaMask's activity, I clicked it and then I had to open another tab to check https://www.ethgasstation.info I went back and the button disappeared and the transaction is still pending it's been 18 hours and I am unable to speed it up.

My guess this happens when there are multiple browser tabs opened. 

I am on Chromium 88.0.4324.192 / macOS Big Sur 11.2.2 / Metamask 9.2.0

**Comment by orion434** [comment]
Same error.
I started the transaction when the suggested slow gas was 4.9 then it got up and I did not have the chance to increase the gas...

![image](https://user-images.githubusercontent.com/36917983/123031722-a76aeb00-d3dc-11eb-81b6-a07b856f61e6.png)
![image](https://user-images.githubusercontent.com/36917983/123031773-c1a4c900-d3dc-11eb-8dae-69184a6b781e.png)

**Comment by FrankvanCaspel** [comment]
I appear to be having this very issue right now. Yesterday I started a transaction with low (too low, it turned out) gas fees. Pretty quickly after that I clicked the 'speed up button' and increased the gas fees, but while doing so I did tab to other windows. I then finished speeding up the transaction, but if I check Etherscan the transaction is still listed with the old gas price, and I can't speed up again because (after hours and restarting browser and logging out of Metamask) the speed-up button is now gone:

![image](https://user-images.githubusercontent.com/17382771/134124483-0b673098-3bec-48cc-83a5-0729c46a5789.png)


I'm on the latest version of Firefox.
